pkg/clair/convert.go

```diff
@@ -220,15 +231,15 @@ func ConvertFeatures(image *storage.Image, features []clairV1.Feature, os string
 	return
 }
 
-// ConvertTime converts a vulnerability time string into a proto timestamp
-func ConvertTime(str string) *timestamp.Timestamp {
+// ParseTime converts a vulnerability time string into nil or a *time.Time object.
+func ParseTime(str string) *time.Time {
```

Comment:
I understand why this was done, but now the clients have to do more work it was hoping to avoid. Can we maybe add this to `protocompat` and do the conversion there, too?

Reply:
I agree with Ross. We should move this function into `protoconv` (not `protocompat`) package where all other time conversion functions live. I think that tests can be moved there too.

And we should not change the function definition for now. We still can return `types.Timestamp` (gogoproto/types).